vm/interpreter/src/default_runtime.rs

```diff
@@ -468,6 +468,10 @@ where
     ) -> Result<(ActorState, Address), ActorError> {
         self.charge_gas(self.price_list().on_create_actor())?;
 
+        if addr.is_bls_zero_address() && self.network_version() >= NetworkVersion::V10 {
+            Err(actor_error!(SysErrIllegalArgument; "cannot create the bls zero address actor"))?
```

Comment:
`ErrIllegalArgument` is used by Lotus so this is a problem. Is this ok in Lotus? I would say Lotus should fix and use`SysErrIllegalArgument` too no?

Reply:
Return directly the error without `?` to make `cargo clippy` happy.

Reply:
updated this!